src/cars/models.py

```diff
@@ -1,3 +1,72 @@
 from django.db import models
 
-# Create your models here.
+
+class Category(models.Model):
+    image = models.ImageField(upload_to="media/cars/categories", null=True, blank=True)
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        verbose_name_plural = "Categories"
+        verbose_name = "Category"
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Brand(models.Model):
+    brand = models.CharField(max_length=20)
+    model = models.CharField(max_length=10)
+    year_of_issue = models.PositiveSmallIntegerField(default=2000, blank=False)

```

Comment:
what is model here? You have model brand so here should be information about brand